src/goose/brokers/redis/commands.clj

```diff
@@ -110,6 +110,9 @@
        (filter match?)
        (doall)))
 
+(defn move-btn-set [conn src dst elem]
```

Comment:
move-btw-set might be better. this reads as button more

Reply:
nitpick: move-between-set**s** ?

Reply:
Jeez! Thanks. 🙈 Will update